plugins/org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/util/IssueUtil.java

```diff
@@ -56,8 +56,8 @@ public Issue createIssue(IMarker marker) {
 				issue.setOffset((Integer) offset);
 				issue.setLength((Integer) endOffset - (Integer) offset); 
 			} else {
-				issue.setOffset(null);
-				issue.setLength(null);
+				issue.setOffset(-1);
```

Comment:
-1 will probably cause other follow-up errors. Which problem did you see with null for Integer?